aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -292,9 +292,18 @@ pub struct AggregatorData {
     destroyed_aggregators: BTreeSet<AggregatorID>,
     // All aggregator instances that exist in the current transaction.
     aggregators: BTreeMap<AggregatorID, Aggregator>,
+    // Counter for generating identifiers for AggregatorSnapshots.
+    pub id_counter: AtomicU64
```

Comment:
Isn't AggregatorData per session only and session is per thread? Then no need to use atomics

Reply:
Done.